targets/javascript/make.js

```diff
@@ -124,14 +142,21 @@ function getUrl(apiCall) {
 }
 
 function getAuthParams(apiCall) {
-    if (apiCall.url === "/Authentication/GetEntityToken")
+    var authConditionArray = [
+        apiCall.url === "/Authentication/GetEntityToken",
+        apiCall.auth === "EntityToken",
+        apiCall.auth === "SecretKey",
+        apiCall.auth === "SessionTicket"
+    ];
+
+    function checkAuthCondition(evaluatedExpression) {
+      return evaluatedExpression === true;
+    }
+
+    // EntityToken, SecretKey, & SessionTicket auth conditions now support authKey and authValue in getRequestActions()
+    // The some() method tests that at least one element in the array passes the test implemented by the provided function
+    if (authConditionArray.some(checkAuthCondition))
```

Comment:
I don't really understand this in the first pass, so I'm probably going to bring it up again in the next review.